nixos/modules/services/web-apps/stirling-pdf.nix

```
@@ -0,0 +1,168 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
```

```suggestion
  options,
  ...
```

to make the suggestion about nginx's vhost type further down work